app/models/user_event.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class UserEvent < ApplicationRecord
+  belongs_to :attendee, class_name: 'User'
+  belongs_to :attended_event, class_name: 'Event'
+
+  validates :attended_event_id, presence: true
```

Comment:
- [ ] `belongs_to` association already adds presence validation. For Rails 5 there is a config named `Rails.application.config.active_record.belongs_to_required_by_default` is set always to `true`, so you don't have to add presence validation for both `belongs_to` associtaions in `friend_request` and `friendship` models [1](https://til.hashrocket.com/posts/ca64beda59-rails-5-changed-belongsto-default-validation).